Gemfile

```diff
@@ -54,9 +54,12 @@ group :development, :test do
   # Add RSpec for testing
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍